### PR TITLE
Added conversion between class and interface node

### PR DESCRIPTION
**Pull request by ghost**
Changed also all special characters in classes to unicodes.

### CODE REVIEW

File: violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/EditorPart.java [hunk 1]
```diff
@@ -181,6 +214,72 @@ public void removeSelected()
...
+				INode convertedNode = switchableNode.switchNode();
+				if (convertedNode != null) {
+					graph.removeNode(node);
+					addNodeAtPoint(convertedNode, node.getLocationOnGraph());
+				} else {
+					JOptionPane.showMessageDialog(this,
+							CONVERT_ERROR_TEXT,
+							CONVERT_ERROR_TITLE,
+						    JOptionPane.ERROR_MESSAGE);
+					System.exit(-1);
```
Comment (abizukojc) [inline violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/EditorPart.java:247]: Why you want to stop the application ?

File: violet-framework/src/main/resources/properties/NodeAndEdgeStrings_de.properties [hunk 2]
```diff
@@ -6,6 +6,13 @@ label.end=Endbeschriftung
 style.line=Linienstil
 style.bent=Kurvenstil
 
+convert.warn.yes=Ja
+convert.warn.no=Nein
+convert.warn.text=Konvertierung kann Methoden und Kommentare aus Klassen beim Konvertieren entfernen. M�chtest du fortfahren?
```
Comment (abizukojc) [inline violet-framework/src/main/resources/properties/NodeAndEdgeStrings_de.properties]: For national characters, use Unicode format \uXXXX

**Review by abizukojc — CHANGES_REQUESTED** [review]
Check the comments

File: violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/EditorPart.java (hunk 1 above)
Comment (ghost) [reply violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/EditorPart.java:247]: Very low level exception catched here - i got instructions to exit app and update log after that. We don't have log in app so i will only stop app with proper information.

**Comment by ghost** [comment]
Updated unicodes.

File: violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/EditorPart.java (hunk 1 above)
Comment (RoughTomato) [reply violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/EditorPart.java:247]: If there's no better way of handling this than okay. Need second opinion though. Stuff builds. So if it also work as it suppose to than maybe we should pass it? deadline is damn close anyway.

Comment (abizukojc) [reply violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/EditorPart.java:247]: Do you know when this exception appears ?

Comment (ghost) [reply violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/EditorPart.java:247]: Exception appears when clone is buggy :) This is "impossible" so it SHOULD never appears :)

Comment (ghost) [reply violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/EditorPart.java:247]: But i got instruction to catch it and add terminate  app - so i did it. Before it was just continue in this loop, so nothing big will happened.

**Review by abizukojc — APPROVED** [review]

**Review by orchowski — APPROVED** [review]

**Pull request merged**